docs/install/windows.md

```diff
@@ -1,37 +1,392 @@
 # Windows build
 
+How to make Neuron compilable in Windows.  
+ 
 ## Pre-requisites
 
-You need to install the following:
+You need to follow the steps:
 
-*  Git: [https://git-scm.com/](https://git-scm.com/)
-*  Chocolatey: [https://chocolatey.org/](https://chocolatey.org/)
-*  Visual Studio with C++: [https://visualstudio.microsoft.com/vs/features/cplusplus/](https://visualstudio.microsoft.com/vs/features/cplusplus/)
-*  `pwsh` (grab the latest PowerShell release on GitHub: [https://github.com/PowerShell/PowerShell/releases/latest](https://github.com/PowerShell/PowerShell/releases/latest))
+### Step 1: (If installing neuron on a Windows Virtual Machine) 
+
+Create a new VM 
+  
+------------------------------------------------------------ 
+
+### Step 2: Install Git 
+
+Git: [https://git-scm.com/](https://git-scm.com/)
+
+Use this: [https://git-scm.com/download/win](https://git-scm.com/download/win)
+
+Note: Do not use: `winget install --id Git.Git -e --source winget` 
+ 
+  
+------------------------------------------------------------ 
+
+### Step 3: Install Chocolatey [https://chocolatey.org/](https://chocolatey.org/) by command line: 
+
```

Comment:
Should probably comment that this should be run in `powershell` or `pwsh`; in particular, it will not work in `cmd`.